submissions/krylenger/dom-api-task/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Motorcycles</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <button class="nav-btn-toggle"></button>
+    <nav class="nav">
+        <ul class="nav__ul">
+            
+        </ul>
+    </nav>
+    <main class="main">
+        <h1 class="main__header"> Choose your bike</h1>
+        <div class="main__img-container">
+            <img src="./images/logo.png" alt="img" class="main__img">
+        </div>
+        <div class="main__description">
+
+        </div>
+        <ul class="main__ul-bikes">
+
+        </ul>
+        <p class="main__avg-price"></p>
+    </main>
+    <script src="./script.js"></script>
+</body>
+
+</html>
```

Comment:
Don't forget about a new line

Reply:
Prettier always "helps" me to remove it)
✅ Done